.github/workflows/playwright-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           node-version: "18"
           cache: "yarn"
       - run: yarn --immutable
-      - run: yarn run bootstrap
+      - run: yarn run build
```

Comment:
All of these "bootstrap" -> "build" changes are fine, but it looks like both just alias `pkg-build` anyway so are the changes necessary?

Reply:
they are not, I just added `bootstrap` as an alias when trying to release this - I mentioned the reason in the comment [here](https://github.com/replayio/replay-cli/pull/549#issuecomment-2189484418)